### PR TITLE
Add rerun_auth_config to kubernetes-e2e-windows-servercore-cache job

### PR DESCRIPTION
**Pull request by spiffxp**
Tried to add teams that would make sense for broad coverage, and also
gave claudibelu direct access.

### CODE REVIEW

File: config/jobs/image-pushing/k8s-staging-e2e-test-images.yaml [hunk 1]
```diff
@@ -40,6 +40,14 @@ periodics:
 # a monthly DLL cache image and use the cache instead.
 # For more info: https://github.com/kubernetes/kubernetes/pull/93889
 - name: kubernetes-e2e-windows-servercore-cache
+  rerun_auth_config:
```
Comment (MushuEE) [inline config/jobs/image-pushing/k8s-staging-e2e-test-images.yaml:43]: Is this still valid?
```
March 19, 2020 The rerun_auth_config config field has been deprecated in favor of the new rerun_auth_configs field which allows configuration on a global, organization or repo level. rerun_auth_config will be removed in July 2020.
```

Comment (MushuEE) [reply config/jobs/image-pushing/k8s-staging-e2e-test-images.yaml:43]: /hold

Comment (spiffxp) [reply config/jobs/image-pushing/k8s-staging-e2e-test-images.yaml:43]: I believe that is in reference to plank's config, not the job config

Comment (MushuEE) [reply config/jobs/image-pushing/k8s-staging-e2e-test-images.yaml:43]: /hold cancel

**Pull request merged**